libraries/apollo-gradle-plugin/src/test/kotlin/com/apollographql/apollo3/gradle/util/TestUtils.kt

```diff
@@ -66,6 +66,15 @@ object TestUtils {
       appendLine("}")
       appendLine()
 
+      append("""
+        java {
+          toolchain { 
+            languageVersion.set(JavaLanguageVersion.of(8))
+          }
+        }
+      """.trimIndent())
```

Comment:
Gradle 8 uses the Gradle JSK as toolchain by default now. See https://slack-chats.kotlinlang.org/t/8496136/i-m-confused-about-https-kotlinlang-org-docs-gradle-configur